apps/web/src/composables/wallet.ts

```diff
@@ -110,15 +113,50 @@ export default function useWallet() {
     }
   }
 
+  async function getUsersPools() {
```

Comment:
In the contract, I went with getPoolsForUser instead to avoid confusion around the name. You might consider using that as well.